src/state/reducers.ts

```diff
@@ -1,10 +1,27 @@
-import {
-  combineReducers, Action, Reducer, ReducersMapObject,
-} from 'redux';
-import query from '../search/state/reducers';
-import results from '../results/state/reducers';
+import { combineReducers, Action, Reducer, ReducersMapObject } from 'redux';
+import query, { SearchAction } from '../search/state/reducers';
+import results, { ResultAction } from '../results/state/reducers';
+import { SearchState } from '../search/state/initialState';
+import { ResultsState } from '../results/state/initialState';
 
-export default combineReducers({
+type RootState =
+  | {
```

Comment:
is this extra `|` there on purpose?

Reply:
For some reason, Prettier adds this, trying to find a workaround....

Reply:
It was in the wrong place.